studio/app/optinist/wrappers/caiman/cnmf.py

```diff
@@ -116,33 +120,42 @@ def caiman_cnmf(
         backend="local", n_processes=None, single_thread=True
     )
 
-    cnm = cnmf.CNMF(n_processes=n_processes, dview=dview, Ain=None, params=ops)
+    cnm = cnmf.CNMF(n_processes=n_processes, dview=dview, Ain=Ain, params=ops)
```

Comment:
@quanpython 
Sorry for the delay in noticing this, but I have observed the following warning when running cnmf-e.

> WARNING:root:using CNMF-E's ringmodel for background hence setting key normalize_init in group init automatically to False.

We would appreciate your investigating this and addressing the issue if it appears to be a problem.

*I'm unclear if this is related, but the following issue described the same log.
- https://github.com/flatironinstitute/CaImAn/issues/521

Reply:
@itutu-tienday
Because the default value of normalize_init is True, when running cnmfe it will automatically convert it to False and display a warning. (https://github.com/flatironinstitute/CaImAn/blob/17cd8e2c11528acde38e750f282fa3774b093f0f/caiman/source_extraction/cnmf/params.py#L935)

This warning will not affect the workflow. To suppress the warning, `normalize_init=False` needs to be added to cnmfe's constant parameter.

Reply:
@quanpython 
As you pointed out, it looks like `normalize_init` needs to be fixed at False.

https://github.com/flatironinstitute/CaImAn/blob/17cd8e2c11528acde38e750f282fa3774b093f0f/demos/notebooks/demo_pipeline_cnmfE.ipynb
> 'normalize_init': False,               # just leave as is

→ Could you add `normalize_init: False` to the fixed parameters?